modules/apps/29-fee/types/fee.go

```diff
@@ -60,8 +60,10 @@ func NewFee(recvFee, ackFee, timeoutFee sdk.Coins) Fee {
 }
 
 // Total returns the total amount for a given Fee
+// The total amount is the Max(TimeoutFee, AckFee + RecvFee)
 func (f Fee) Total() sdk.Coins {
-	return f.RecvFee.Add(f.AckFee...).Add(f.TimeoutFee...)
+	// maximum returns the denomwise maximum of two sets of coins
+	return f.RecvFee.Add(f.AckFee...).Max(f.TimeoutFee)
```

Comment:
Can we add unit tests for this? Just want to make sure this doesn't accidentally become: `recvFee.Add(max(ack, timeout))`

Reply:
There is a unit test, I will expanded it though.

Reply:
Can you add these comments under the unit tests for Total. I thought my example was sufficient but I can add more test cases if need be.